### DISCUSSION TITLE
Why we didn't approached PWA approch

### DISCUSSION THREAD

**Discussion by ghost** [ReVanced/revanced-discussions#964] [Question] [answered]
I want to ask that why didn't we approached the vanced access of yt by PWA( Progressive Web App ) , itll be much much simpler than dealing with microG

Itll be light weight & highly customisable

similar web frontend is already currently available named piped.video by teampiped grate effort kudos for them

but i suggest we should now approch the problem with PWA & by doing so users ( * in future ) just hav to dl a zip of new patch , and they will be ready to roll link in seconds , because PWA works on top of web browsers and have some sufficient isolation to be used without breking into users privacy.

Best part mods will spend less time dealing with users and more time devloping the app & apis

Thank you for your time & attention

**Comment by oSumAtrIX** [comment, marked as answer]
Because YouTube is not web optimized.

**Reply by ghost** [reply]
not optimised as in which part

**Comment by reisxd** [comment]
ReVanced is a patcher, not an alternative of YT with no ads, etc.
